CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 
 ### Bug fixes
 
+* [#7338](https://github.com/rubocop-hq/rubocop/issues/7338): Allow using indentation for variable's methods after assignment. ([@neodelf][])
```

Comment:
Looking at the issue report this description seems a bit misleading - it seem the bug exists only if you have `[]` on the lhs.

Reply:
Is `Allow using indentation for variable's methods if you have [] on the LHS.` would be better?

Reply:
Outdated